src/activate_linux.c

```diff
@@ -108,11 +123,14 @@ int main(int argc, char *argv[]) {
     // bypass compositor hint
     bool bypass_compositor = false;
 
+    // screen to display to as a bit mask, up to 32 screen supported
+    uint32_t display_screen = 0xFFFFFFFF;
```

Comment:
Last moment I see is this symbol. I'd prefer to rename this to something like `screens_mask` or anything showing plural number and selectivity :)

Reply:
That's a much better name. I am so so bad at finding meaningful names for variables

Reply:
Thank you! ^^